storage/src/test/java/tech/pegasys/teku/storage/client/RecentChainDataTest.java

```diff
@@ -798,7 +798,9 @@ public void commit_pruneParallelExistingBlocks() {
   @Test
   public void getAncestorsOnFork() {
     initPreGenesis();
-    final ChainBuilder chainBuilder = ChainBuilder.create(BLSKeyGenerator.generateKeyPairs(16));
+    final ChainBuilder chainBuilder =
+        ChainBuilder.create(
+            TestSpecFactory.createMinimalPhase0(), BLSKeyGenerator.generateKeyPairs(16));
```

Comment:
Can use the existing `spec` field:
```suggestion
            spec, BLSKeyGenerator.generateKeyPairs(16));
```